homeassistant/components/home_connect/api.py

```diff
@@ -371,14 +393,14 @@ class WasherDryer(
 
     def get_entity_info(self):
         """Get a dictionary with infos about the associated entities."""
-        door_entity = self.get_door_entity()
+        door_entities = self.get_door_entities()
```

Comment:
Potentially remove variables for this entire section?  (Maybe a bit beyond the scope of the PR)

Reply:
Yeah, this one feels a bit beyond scope as this integration seems to use single use variables throughout. I wasn't trying to change too much code for my first PR. 😄